src/app/features/admin/services/popup.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PopUpService {
+
+  makeCapitalPopup(title:string): string {
+    return (
+      `` +
+      `<div>Station: ${title}</div>`
+    );
+  }
```

Comment:
Why we need to use this service instead of snackbars in Angular Material?

Reply:
It's not about snackbars. It's for showing city name cloud from second screenshot